content/docs/codebase-overview.md

```diff
@@ -149,78 +149,80 @@ var ReactHostComponent = {
+Incluso los renderizadores como React DOM y React Native necesitan compartir gran cantidad de lógica. En particular, el algoritmo de [reconciliación](/docs/reconciliation.html) debe ser tan similar como sea posible para que el renderizado declarativo, los componentes personalizados, el estado, los métodos del ciclo de vida, y refs funcionen de forma consistente a tráves de las plataformas.
 
-To solve this, different renderers share some code between them. We call this part of React a "reconciler". When an update such as `setState()` is scheduled, the reconciler calls `render()` on components in the tree and mounts, updates, or unmounts them.
+Para resolver esto, diferentes renderizadores comparten parte del código. Llamamos a esta parte de React un `reconciliador`. Cuando se planifica una actulización como `setState()`, el reconciliador llama el método `render()` en los componentes del árbol y los monta, actuliza, o desmonta.
 
-Reconcilers are not packaged separately because they currently have no public API. Instead, they are exclusively used by renderers such as React DOM and React Native.
+Los reconciliadores no estan enpaquetados por separado porque actualmente no tienen una API pública. Por el contrario, son exclusivamente usados por los renderizadores como React DOM y React Native.
 
-### Stack Reconciler {#stack-reconciler}
+### Reconcilador de pila {#stack-reconciler}
 
-The "stack" reconciler is the implementation powering React 15 and earlier. We have since stopped using it, but it is documented in detail in the [next section](/docs/implementation-notes.html).
+El reconciliador de "pila" es la implementación que permite el funcionamiento de React 15 y versiones previas. Nosotros paramos de usarlo, pero esta documentada en detalla en la [próxima sección](/docs/implementation-notes.html).
 
-### Fiber Reconciler {#fiber-reconciler}
+### Reconciliador Fiber {#fiber-reconciler}
 
-The "fiber" reconciler is a new effort aiming to resolve the problems inherent in the stack reconciler and fix a few long-standing issues. It has been the default reconciler since React 16.
+El reconciliador "fiber" es un nuevo esfuerzo dedicado a resolver los problemas inherentes al reconciliador de pila y arreglar algunos issues. Ha sido el reconciliador por defecto desde React 16.
 
-Its main goals are:
+Sus objetivos principales son:
 
-* Ability to split interruptible work in chunks.
-* Ability to prioritize, rebase and reuse work in progress.
-* Ability to yield back and forth between parents and children to support layout in React.
-* Ability to return multiple elements from `render()`.
-* Better support for error boundaries.
+// TODO: revizar con atención estos puntos
```

Comment:
Elimina también este comentario, creo que es extra, la línea completa 💪

Reply:
yeah, that was already done, but thanks for the advice!